t/plugin/plugin.t

```diff
@@ -306,3 +306,139 @@ GET /hello
+
+
+
+=== TEST 11: invalid _meta filter schema with wrong expr
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+
+            for _, vars in ipairs({
+                {"arg_name", "==", "json"},
+                {
+                    {"arg_name", "*=", "json"}
+                }
+            }) do
+                local code, body = t('/apisix/admin/plugin_configs/1',
+                    ngx.HTTP_PUT,
+                    {
+                        plugins = {
+                            ["jwt-auth"] = {
+                                _meta = {
+                                    filter = {
+                                        vars = vars
+                                    }
+                                }
+                            }
+                        }
+                    }
+                )
+                if code >= 300 then
+                    ngx.print(body)
+                else
+                    ngx.say(body)
+                end
+            end
+        }
+    }
+--- response_body
+{"error_msg":"failed to validate the 'vars' expression: rule should be wrapped inside brackets"}
+{"error_msg":"failed to validate the 'vars' expression: invalid operator '*='"}
+
+
+
+=== TEST 12: proxy-rewrite plugin run with _meta filter vars
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                ngx.HTTP_PUT,
+                {
+                    plugins = {
+                        ["proxy-rewrite"] = {
+                            _meta = {
+                                filter = {
+                                    vars = {
+                                        {"arg_version", "==", "v2"}
+                                    }
+                                }
+                            },
+                            uri = "/echo",
+                            headers = {
+                                ["X-Api-Version"] = "v2"
+                            }
+                        }
+                    },
+                    upstream = {
+                        nodes = {
+                            ["127.0.0.1:1980"] = 1
+                        },
+                        type = "roundrobin"
+                    },
+                    uri = "/hello"
+                }
+            )
+            if code >= 300 then
+                ngx.print(body)
+            else
+                ngx.say(body)
+            end
+        }
+    }
+--- response_body
+passed
+
+
+
+=== TEST 13: hit route: run proxy-rewrite plugin
+--- request
+GET /hello?version=v2
+--- response_headers
+x-api-version: v2
+--- no_error_log
```

Comment:
It is already set: https://github.com/apache/apisix/blob/ea6b55db7d50133c6cf6adb32539223c45cb0d38/t/plugin/plugin.t#L22